frontend/src/config/myanmar/layers.json

```diff
@@ -1,11 +1,11 @@
 {
   "state_admin_boundaries": {
     "type": "boundary",
-    "path": "data/myanmar/mmr_admin1_boundaries.json",
```

Comment:
Same as the other transfers, maybe delete the unused boundary files?

Reply:
good catch. removed in the last commit